GeneralsMD/Code/GameEngine/Include/GameClient/ClientInstance.h

```diff
@@ -39,6 +39,10 @@ class ClientInstance
 	// Returns the instance name of the first game client.
 	static const char* getFirstInstanceName();
 
+public:
+	static Bool s_multiInstance; // Allow multiple instances of the game
```

Comment:
This is wrong. These variables must not be writable after initialization is done. I have written the functions that need to be added in a previous comment. If you do not have time to do it, I can do that for you.

Reply:
Fixed